### ISSUE TITLE
Test: training loss curve of a minimalistic custom Llama model against PyTorch

### ISSUE THREAD

**Issue by Muxas** [nntile/nntile#126]
Pytest-based solution shall compare training loss curve of a randomly PyTorch-initialised Llama model on a simple text data.
